### ISSUE TITLE
Update water rights variable for AZ data

### ISSUE THREAD

**Issue by amabdallah** [WSWCWaterDataExchange/WestDAAT#66]
Should the variable here be: Allocation? It seems that's the case for all of AZ's data

https://westdaatqa.azureedge.net/details/right/1626514
![image](https://user-images.githubusercontent.com/3268971/161344877-ba0200be-a970-4e50-bf58-efb0fa2b8e6f.png)

BTY, this water right is very interesting!

**Comment by rwjam** [comment]
Looks like that was just how it was mapped out.  If that is not correct we can change it, but would need guidance on what it should be

![image](https://user-images.githubusercontent.com/17555073/161345740-47244d7e-e777-4ac6-b107-b9b7a2039806.png)

**Comment by amabdallah** [comment]
Yep, let's change it but first let's rethink what we call the variable name for all water rights. A few terms I've seen:
Allotment, permit, water right, allocation. Let's ask Michelle and Tony.

**Comment by rwjam** [comment]
Made that change to **Allocation**.
